### DISCUSSION TITLE
Seasonal electricity rates

### DISCUSSION THREAD

**Discussion by geedoes** [teslamate-org/teslamate#812] [Ideas]
**What would you like to be added**:
An option to enter different electricity rates for March through October (Midwest Summer) etc. The ability to enter the electrical company schedule. 

**Why is this needed**:
It would help paint a more accurate picture of what charging costs if you have different rates in the summer versus winter. 

**Additional context**:

Example:
$.18 per kwh for 1/2 the year
$.12 per kwh for other half. 

Thank you so much for TeslaMate. It's been very useful!

**Comment by RonSpawnson** [comment]
Many providers near me have different rates in winter and summer. This would be a very helpful feature. How are others managing this until support is added?

**Comment by metaben** [comment]
When my electricity rate changes (because of market fluctuations, or changing provider), I change the setting manually. You could do that once in march and once in October.

I think seasonal rates might be an American thing.

**Comment by RonSpawnson** [comment]
Can the electricity rates be changed programmatically? Trying to think of a way I can automate this so I don't have to remember to log in and manually change it every 6 months.
